Frontend/android-native/mobility-customer/src/main/assets/juspay/v1-configuration.js

```diff
@@ -19,7 +19,10 @@ window.getMerchantConfig = function () {
     "DOCUMENT_LINK": "https://docs.google.com/document/d/1-oRR_oI8ncZRPZvFZEJZeCVQjTmXTmHA",
     "APP_LINK": "",
     "PRIVACY_POLICY_LINK": "https://docs.google.com/document/d/128VU80K5E1iz-x6QnP1R127m_lwmDO3F",
-    "CUSTOMER_TIP": "false",
+    "CUSTOMER_TIP" : {
```

Comment:
can you update these values in CommonConfig.purs and update required v1-Configurations according